src/Database/PG/Query/Pool.hs

```diff
@@ -7,7 +7,8 @@
 
 module Database.PG.Query.Pool
   ( ConnParams (..)
-  , PGPool
+  , PGPool(..)
+  , PGPoolStats(..)
```

Comment:
I would rather keep the `PGPool` hidden and provide a function `PGPool -> PGPoolStats`